### PR TITLE
Letter 'i' from #436

### PR DESCRIPTION
**Pull request by luzpaz**
Continuing to separate ~#428~ #436 in to separate PRs for each letter. Also removed/corrected some entries that were not relevant/precise and augmented others. This PR has been sorted as well.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -5807,13 +5831,19 @@ inablility->inability
 inacccessible->inaccessible
 inaccesible->inaccessible
 inaccessable->inaccessible
+inactivate->inactive
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: This one is an acceptable verb, so it should stay the first suggestion. You can also suggest "deactivate".

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -5857,10 +5890,12 @@ incomptetent->incompetent
 incomptible->incompatible
 inconsisent->inconsistent
 inconsisently->inconsistently
+inconsistancy->inconsistency
 inconsistant->inconsistent
 inconsisten->inconsistent
 inconsistenly->inconsistently
 inconsitent->inconsistent
+inconstent->inconsistent
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: also "inconstant"

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -6043,38 +6107,51 @@ initally->initially
 initation->initiation
 initators->initiators
 initiaitive->initiative
+initiales->initialize, initializes
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: comma at end

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -6084,21 +6161,32 @@ initilizing->initializing
...
+inluding->including
 inly->only, inly,
 inmigrant->immigrant
 inmigrants->immigrants
+inmmediately->immediately
+inmplementation->implementation
 innacurate->inaccurate
 innacurately->inaccurately
+inneffectual->ineffectual
+innitialize->initializes
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: initialize

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -6319,29 +6471,40 @@ intrumented->instrumented
...
 invertibrates->invertebrates
+inverval->interval
 investingate->investigate
 invididual->individual
 invidual->individual
 invidually->individually
+invisiable->invisible
+invlaid->invalid
+invlisible->invisible
+invokable->invocable
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: invokable seems okay

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -5733,23 +5743,32 @@ implemementations->implementations
...
 implementaitons->implementations
+implementantions->implementations
+implementastion->implementation
 implementated->implemented
 implementatin->implementation
 implementatins->implementations
+implementatition->implementation
+implementatoins->implementations
+implementatoion->implementation
+implemente->implement
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: combine with line below

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: @larsoner please clarify what you mean, thanks.

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: Oh nvm, i see it

**Comment by luzpaz** [comment]
Rebased and ready for merge

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -6075,38 +6138,51 @@ initally->initially
...
+initialitze->initialize
+initialitzing->initializing
+initializaing->initializing
+initializeing->initializing
 initializiation->initialization
-initialse->initialize
-initialsed->initialized
-initialses->initializes
+initializong->initializing
+initialse->initialize, initialize,
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: did you mean `initialise` for the second one?

https://travis-ci.org/lucasdemarchi/codespell/jobs/376840463

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: <facepalm>

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -6075,38 +6138,51 @@ initally->initially
...
 initialzes->initializes
 initialzing->initializing
-initiliase->initialize
-initiliased->initialized
-initiliases->initializes
-initiliasing->initialising
+initiliase->initialise, initialize,
+initiliased->initialised, initialized,
+initiliases->initialises, initializes,
+initiliasing->initialising, initialising,
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: duplicate

https://travis-ci.org/lucasdemarchi/codespell/jobs/376867129

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -6075,38 +6138,51 @@ initally->initially
...
-initilise->initialize
-initilised->initialized
-initilises->initializes
-initilising->initialising
+initilisation->initialisation, initialization,
+initilisations->initialisations, initializations,
+initilise->initialise, initialize,
+initilised->initialised, initialized,
+initilises->initialises, initializes,
+initilising->initialising, initialising,
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: duplicate

File: codespell_lib/data/dictionary.txt [hunk 10]
```diff
@@ -6175,25 +6278,41 @@ instansiate->instantiate
...
 instuction->instruction
 instuments->instruments
 instutionalized->institutionalized
 instutions->intuitions
+insufficent->insufficient
+insuffiency->insufficiency
 insurence->insurance
+intaces->instance
 intall->install
+intance->instance
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: `intance` already exists

https://travis-ci.org/lucasdemarchi/codespell/jobs/376870882#L670

It's probably worth running `pytest codespell_lib` locally so you don't have to wait for Travis to give this feedback

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: how do I run `pytest codespell_lib` ?  
`bash: pytest: command not found`

Comment (larsoner) [reply codespell_lib/data/dictionary.txt]: you need to install the `pytest` and `pytest-cov` Python packages

**Pull request merged**